logrotate.c

```diff
@@ -1914,7 +1914,12 @@ static int rotateLogSet(struct logInfo *log, int force)
     struct logState **state;
     struct logNames **rotNames;
 
-    logHasErrors = alloca(log->numFiles * sizeof(int));
+    logHasErrors = calloc(sizeof(int), log->numFiles);
```

Comment:
Looks good.  The swap of operands is unnecessary though.  The prototype of ```calloc()``` is:
```
void *calloc(size_t nmemb, size_t size);
````
No need to resubmit, of course.